musicHelper.md

```diff
@@ -0,0 +1,128 @@
+#Piano Music Helper
+
+###_Introduction_
```

Comment:
Please remove underscores, it is hard to read

Reply:
Ok, did it